### ISSUE TITLE
(googlechrome) Installation Fails

### ISSUE THREAD

**Issue by infraweavers** [chocolatey-community/chocolatey-coreteampackages#1608]
## Expected Behavior

Google Chrome should install

## Current Behavior

The installer blows up, initially with a checksum change (as happens from time to time) however it then errors completely. 
![image](https://user-images.githubusercontent.com/36475663/105149596-9b13f680-5afb-11eb-89cc-22d7b7578908.png)
[GoogleChrome_msi_install.log](https://github.com/chocolatey-community/chocolatey-coreteampackages/files/5841131/GoogleChrome_msi_install.log)

## Possible Solution

I'm not sure yet, presumably the Google Chrome installer has been changed and this has broken it?

## Steps to Reproduce (for bugs)

1. `choco install googlechrome --ignore-checksums --install-arguments="'/l*v c:\GoogleChrome_msi_install.log'"`

## Context

Can't install GoogleChrome

## Your Environment

* Package Version used: GoogleChrome v87.0.4280.141
* Operating System and version: Windows 10 20H2
* Chocolatey version: Chocolatey v0.10.15
* Install/uninstall gist:

**Comment by david-garcia-garcia** [comment]
Exatly same issue here. Happening since today. Same log and errors.

**Comment by tariqp193** [comment]
Same issue here.

**Comment by vkmellon** [comment]
We have the same

**Comment by tomworboys** [comment]
Same here as of this morning

**Comment by Greg-Smulko** [comment]
Please use the 👍 emoji on the main comment rather than posting "me too" comments.
This way you're saying the same thing basically, but without spamming everybody who subscribed to this issue waiting for a resolution. 
Thanks! :) 

PS: link to a discussion about it: https://github.com/isaacs/github/issues/968

**Comment by AdmiringWorm** [comment]
I looked a bit into this issue to try finding out what is going on.
From what I can tell at the moment, the installer acquired from the website is basically broken, and refuses to install even outside of chocolatey.
![image](https://user-images.githubusercontent.com/1474648/105181573-42a91d00-5b2c-11eb-81aa-bab5bee65f39.png)


I also tried some other standalone installers found on the website, but these had the same issue.
As such, it isn't much we can do unfortunately and can only wait on Google to update the installer with one that isn't broken.

**Comment by jerasioren** [comment]
Exact same error on my end.
Was able to install it manually but it was for the latest version of 88.0.4324.96

**Comment by akrapfl** [comment]
> Please use the 👍 emoji on the main comment rather than posting "me too" comments.
> This way you're saying the same thing basically, but without spamming everybody who subscribed to this issue waiting for a resolution.
> Thanks! :)
> 
> PS: link to a discussion about it: [isaacs/github#968](https://github.com/isaacs/github/issues/968)

Not intending to start an argument, but I did want to point out that having SOME comments in the thread today made it clear this is a current and legit issue on the installer side, and not something on MY side.

Thanks for posting the issue!

**Comment by infraweavers** [comment]
@AdmiringWorm I wonder if there's a way of reporting packaging problems to Google somehow? I can't find anything particular, however my GoogleFoo may be weak

**Comment by AdmiringWorm** [comment]
@infraweavers unfortunately, I do not know of any way to report the problem to Google.
Just to clarify on the terminology a bit though, it would be an installer problem, not a package problem.

**Comment by AdmiringWorm** [comment]
small update, I tried to install the chrome package on a different VM which somehow succeeded.
So, it could be that there are some changes to the installer that checks for some missing dependencies and throws an error when these are not found.

Still investigating what these dependencies could possibly be though.

**Comment by AdmiringWorm** [comment]
Have to give up for today, but hope to try looking into this issue further tomorrow.

I am attaching the latest log that I got from the installer (a bit more verbose than the one that OP posted), in case someone want to continue looking into this.
From what I could tell, the log does not contain anything useful but it could be that I am missing something.

[MSI41905.LOG](https://github.com/chocolatey-community/chocolatey-coreteampackages/files/5844838/MSI41905.LOG)

**Comment by jcrang** [comment]
In our use case (tests run with Selenium), we were able to use [Chromium](https://chocolatey.org/packages/chromium/88.0.4324.96) as a drop-in replacement.

**Comment by tgillitzer** [comment]
Same issue here.   I found this report on the Google Chrome Enterprise Help community
https://support.google.com/chrome/a/thread/94282332?hl=en

I also submitted an issue with Chrome itself, though that doesn't seem likely to work.

**Comment by AdmiringWorm** [comment]
@tgillitzer that issue would be the one to follow, as the package also makes use of the enterprise installer (this is the only standalone installer available, the default chrome one downloads a different installer during launch).

I have given up on my tries to make it work, but that issue explains one reason I got it to work on one of the vms I tried it one (it had v87 installed earlier).

**Comment by AdmiringWorm** [comment]
Here is another issue, also with a response from the community managers: https://support.google.com/chrome/a/thread/94236509?hl=en

looks like they have identified the root cause and is working on a fix.

**Comment by tgillitzer** [comment]
This seems to be the link to the actual bug report

https://bugs.chromium.org/p/chromium/issues/detail?id=1168614

**Comment by j30a4** [comment]
Looks like they just pushed out a new version with the fix M88.0.4324.104 and MSI seems to work now. 

https://bugs.chromium.org/p/chromium/issues/detail?id=1168614

**Comment by AdmiringWorm** [comment]
Good to know. Our updater will automatically trigger in ~30 minutes and will automatically pick up the new version then.

It will take a few hours after that until it will be available for normal consumption through chocolatey.org.

**Comment by AdmiringWorm** [comment]
A new package version was pushed and approved last night which works as expected, as such I will be closing this issue now.
